controllers/operatorscaler_controller.go

```diff
@@ -47,19 +48,21 @@ const (
 )
 
 type ResourceMappingRecord struct {
-	CrdName    string
-	ApiVersion string
-	Kind       string
-	PkgNames   []string
+	CrdName        string
+	ApiVersion     string
+	Kind           string
+	PkgNames       []string
+	RemoveOwnerRef bool
```

Comment:
as mentioned in another PR, this new addition is internal to operator logic and should be separated.